src/NUnitFramework/tests/nunit.framework.tests-3.5.csproj

```diff
@@ -122,6 +123,7 @@
     <Compile Include="Internal\Results\TestResultOutputTests.cs" />
     <Compile Include="Internal\Results\TestResultSuccessTests.cs" />
     <Compile Include="Internal\Results\TestResultTests.cs" />
+    <Compile Include="Internal\Results\TestResultWarningTests.cs" />
```

Comment:
This is unrelated - just noticed this file was missing from all except the 2.0 build.